sorc/ncep_post.fd/PARA_RANGE.f

```diff
@@ -46,4 +46,42 @@ SUBROUTINE PARA_RANGE (N1,N2,NPROCS,IRANK,ISTA,IEND)
       if ( iwork2 > irank ) iend = iend + 1
       return
       end
+!!
+!! USAGE:    CALL PARA_RANGE2(N1,N2,NX,NY,NRANK,ISTA,IEND,JSTA,JEND)(A)
+!!   INPUT ARGUMENT LIST:
+!!     N1 - LAAT INTERATE VALUE I dimension
+!!     N2 - LAST INTERATE VALUE  J dimension
+!!     NX  NUMBER OF subdomains in Z  dimension
+!!     NY  NUMBER OF subdomains  in Y dimension
+!!        NX * NY should be the total number of MPI procs
+!!     NRANK - MY TAKS ID
+!!
+!!   OUTPUT ARGUMENT LIST:
+!!     ISTA - FIRST LOOP VALUE I
+!!     IEND - LAST LOOP VALUE I
+!!     JSTA - FIRST LOOP VALUE J
+!!     JEND - LAST LOOP VALUE J
+!!
+!!   OUTPUT FILES:
+!!     STDOUT  - RUN TIME STANDARD OUT.
+!!
+!!   SUBPROGRAMS CALLED:
+!!     UTILITIES:
+!!       NONE
+!!     LIBRARY:
+!!
+!!   ATTRIBUTES:
+!!     LANGUAGE: FORTRAN
+!!     MACHINE : IBM RS/6000 SP
+!!
+     subroutine para_range2(im,jm,nx,ny,nrank,ista,iend,jsta,jend)
```

Comment:
@GeorgeVandenberghe-NOAA @JesseMeng-NOAA @BoCui-NOAA Would variable declarations be needed in this subroutine?

Reply:
It should. Do you want me to add it?

Reply:
@JesseMeng-NOAA Please update it. Thanks!